docker-compose-collator/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# ===== Rust builder =====
+FROM phusion/baseimage:focal-1.1.0 as builder
```

Comment:
Can use [composablefi/ci-linux:production](https://github.com/ComposableFi/composable/blob/main/docker/ci-linux.dockerfile) as builder. It has all deps to build collator.

Reply:
@alexkdefi can we use this` composablefi/ci-linux:production` instead of  this `phusion/baseimage:focal-1.1.0` as base image?